graphql/resolvers.go

```diff
@@ -1886,6 +1886,23 @@ func (r *mutationResolver) MoveAnnotationIssue(ctx context.Context, annotationID
 	}
 }
 
+func (r *mutationResolver) AddAnnotationIssue(ctx context.Context, taskID string, execution int,
```

Comment:
I feel like we might benefit from a comment somewhere that explains what the isIssue boolean is for, because I don't think it's immediately apparent that it can be an issue or suspected issue, and this serves as the differentiation. I don't know where the appropriate place for that explanation would be though.

Reply:
Good point! I added function comments for these resolver functions.